.github/workflows/patrol-publish.yaml

```diff
@@ -5,9 +5,63 @@ on:
     tags: ['patrol-v*']
 
 jobs:
+  get_last_released_version:
+    name: Get last released version
+    runs-on: ubuntu-latest
+    outputs:
+      last_version: ${{ steps.get_last_released_version.outputs.last_version }}
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get last released version
+        id: get_last_released_version
+        run: |
+          last_version=$(git tag --list 'patrol-v*' | grep -v '\-dev\.[0-9]\+$' | sort -V | tail -n 1 | sed 's/^patrol-v//')
+          echo "last_version=$last_version" >> $GITHUB_ENV
+          echo "::set-output name=last_version::$last_version"
+
+  semver:
+    name: Check API changes
+    runs-on: ubuntu-latest
+    needs: [get_last_released_version]
+
+    strategy:
+      matrix:
+        flutter-version: ['3.22.x']
+        flutter-channel: ['stable']
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+
+      - name: Set up Dart
```

Comment:
Do you really need to setup dart separately?

Reply:
I'm pretty sure it's not needed :D